manifest.php

```diff
@@ -30,7 +30,7 @@
     'label'       => 'Browser and OS diagnostic tool',
     'description' => 'Check compatibility of the os and browser of a client',
     'license'     => 'GPL-2.0',
-    'version'     => '7.3.4',
+    'version'     => '7.3.3.1',
```

Comment:
7.3.4.1?

Reply:
The version that this update is coming from was originally 7.3.3. I wanted to update this to 7.3.4, but just saw there is already a tag like that. So as this release is based on 7.3.3, it makes sense to go to 7.3.3.1